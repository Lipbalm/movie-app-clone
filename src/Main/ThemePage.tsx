import React, { FC } from "react";
import { BodyWrapper } from "./MainStyle";
const ThemePage: FC = () => {
  return (
    <BodyWrapper>
      <div>ThemePage</div>
    </BodyWrapper>
  );
};

export default ThemePage;
