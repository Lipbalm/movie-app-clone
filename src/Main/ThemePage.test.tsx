import React from "react";
import { render, fireEvent } from "@testing-library/react";
import ThemePage from "./ThemePage";

function setup() {
  return { ...render(<ThemePage />) };
}

describe("<ThemePage />", () => {
  it("snapshot", () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
