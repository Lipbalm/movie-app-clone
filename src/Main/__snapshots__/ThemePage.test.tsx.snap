// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemePage /> snapshot 1`] = `
<div>
  <div
    class="sc-ifAKCX cktARQ"
  >
    <div>
      ThemePage
    </div>
  </div>
</div>
`;
